test/language/literals/string/legacy-non-octal-escape-sequence-strict.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2017 Kevin Gibbons. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-additional-syntax-string-literals
+es6id: B1.2
+description: >
+    LegacyOctalEscapeSequence is not enabled in strict mode code (regardless of
+    the presence of Annex B)
+info: >
+    EscapeSequence ::
+      CharacterEscapeSequence
+      LegacyOctalEscapeSequence
+      HexEscapeSequence
+      UnicodeEscapeSequence
+
+    LegacyOctalEscapeSequence ::
+      OctalDigit [lookahead ∉ OctalDigit]
+      ZeroToThree OctalDigit [lookahead ∉ OctalDigit]
+      FourToSeven OctalDigit
+      ZeroToThree OctalDigit OctalDigit
+
+    ZeroToThree :: one of
+      0 1 2 3
+
+    FourToSeven :: one of
+      4 5 6 7
+
+    This definition of EscapeSequence is not used in strict mode or when
+    parsing TemplateCharacter.
+flags: [onlyStrict]
+negative:
+  phase: early
+  type: SyntaxError
+---*/
+
+'\08';
```

Comment:
@littledan Right. But the more normal cases are already covered [here](https://github.com/tc39/test262/blob/0314c87b0816e78802533ba9b30d42fb30eebf19/test/language/literals/string/legacy-octal-escape-sequence-strict.js) and [here](https://github.com/tc39/test262/blob/0314c87b0816e78802533ba9b30d42fb30eebf19/test/language/literals/string/legacy-octal-escape-sequence-strict.js). Or was there something not covered by those two + this one you thought ought to be tested?